Editor/AssetBundleBrowserMain.cs

```diff
@@ -41,9 +52,9 @@ enum Mode
         [MenuItem("Window/AssetBundle Browser", priority = 2050)]
         static void ShowWindow()
         {
-            var window = GetWindow<AssetBundleBrowserMain>();
-            window.titleContent = new GUIContent("AssetBundles");
-            window.Show();
+            m_instance = GetWindow<AssetBundleBrowserMain>();
```

Comment:
if you are wrapping m_instance in a property that will call GetWindow if null, just use the property

Reply:
that just blew my mind.  good call.